package Ejercicio_31_excepciones_perosnales;

public class Division {
	
	
	
	
	public static void main(String [] args) {
		
		try {
		validar.esValido(10<=4);
		}catch(MiError e) {
		System.out.println(e.getMessage());
		}
		
	}

}
